src/connect.rs

```diff
@@ -905,7 +906,14 @@ mod verbose {
             cx: &mut Context<'_>,
             bufs: &[IoSlice<'_>],
         ) -> Poll<Result<usize, io::Error>> {
-            Pin::new(&mut self.inner).poll_write_vectored(cx, bufs)
+            match Pin::new(&mut self.inner).poll_write_vectored(cx, bufs) {
+                Poll::Ready(Ok(nwritten)) => {
+                    log::trace!("{:08x} write: {:?}", self.id, Vectored { bufs, nwritten });
```

Comment:
Just thinking, would it be useful if the log indicated it was a writev? Or "write (vectored)" if people are filtering by "write"?